exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/metadata/Metadata.java

```diff
@@ -101,14 +102,15 @@ private Metadata(ParquetReaderConfig readerConfig) {
 
   /**
    * Create the parquet metadata file for the directory at the given path, and for any subdirectories.
-   *
    * @param fs file system
    * @param path path
    * @param readerConfig parquet reader configuration
+   * @param allColumns
```

Comment:
Please add description to params to avoid warnings in the IDE.

Reply:
Added description